src/client/replication_ddl_client.cpp

```diff
@@ -1547,5 +1549,36 @@ void replication_ddl_client::query_disk_info(
     call_rpcs_async(query_disk_info_rpcs, resps);
 }
 
+error_with<start_bulk_load_response>
+replication_ddl_client::start_bulk_load(const std::string &app_name,
+                                        const std::string &cluster_name,
+                                        const std::string &file_provider_type)
+{
+    auto req = make_unique<start_bulk_load_request>();
+    req->app_name = app_name;
+    req->cluster_name = cluster_name;
```

Comment:
When connect to a cluster to start bulk load, the cluster name is definite, so why specify the cluster name?

Reply:
The cluster name is used to find sst files on file provider(fds), the files are under the path: `<bulk_load_provider_root>/<cluster_name>/<app_name>`